lib/ansible/modules/windows/win_psmodule.ps1

```diff
@@ -69,7 +69,7 @@ Function Install-PrereqModule {
             }
             else {
                 try {
-                    Install-Module -Name $Name -MinimumVersion $PrereqModules[$Name] -Force -WhatIf:$CheckMode | Out-Null
+                    Install-Module -Name $Name -MinimumVersion $PrereqModules[$Name] -Force -SkipPublisherCheck -WhatIf:$CheckMode | Out-Null
```

Comment:
Possibly irrelevant, but looking here: https://docs.microsoft.com/en-us/powershell/module/powershellget/install-module?view=powershell-5.0 it appears -SkipPublisherCheck was not present before powershell 5.1, so just wondering if this needs handling for older powershell versions?

Reply:
"Install-Module runs on PowerShell 5.0 or later releases, on Windows 7 or Windows 2008 R2 and later releases of Windows." https://docs.microsoft.com/en-us/powershell/module/powershellget/install-module?view=powershell-6#notes

Reply:
I think you are right, should we require 5.1 and above or add a conditional?

Reply:
Added a check